engine/lib/river.php

```diff
@@ -365,6 +369,9 @@ function elgg_get_river(array $options = array()) {
 		}
 	}
 
+ 	// add optional joins
```

Comment:
how are the space characters denoted in github? i am not seeing them... these lines look exactly the same as all the other lines for me in github's diff view. they have a plus sign at the start and then just empty space up until where the text begins.